runtime/amd64.S

```diff
@@ -82,7 +82,9 @@
 #endif
 
 #ifdef ASM_CFI_SUPPORTED
-#define CFI_STARTPROC .cfi_startproc
+#define CFI_STARTPROC \
+        .cfi_startproc; \
+        endbr64
```

Comment:
Same comment as above. That should be in `FUNCTION`, not `CFI_STARTPROC`.

Reply:
Moved to the `FUNCTION` define